### DISCUSSION TITLE
Flow Calibration small change

### DISCUSSION THREAD

**Discussion by MrExo3D** [OrcaSlicer/OrcaSlicer#435] [Ideas]
the current flow calibration test is great, but it would be even better if each test showed the actual flow used. 0.99, 1.01 etc instead of +1, 0, -4  etc .

faster to adjust, less math mistakes,

**Comment by Mario-vminion** [comment]
when you click objects next to global you can see the values of the flow what is used ... easy to see and adjust

**Reply by MrExo3D** [reply]
yep but even easier if its written out in the test print

**Reply by Mario-vminion** [reply]
You can do that with the build in 3d text option
